lib/ui/dashboard_page.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+class DashboardPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Container(
+        child: const Text(
+          "Dashboard Page",
```

Comment:
same as above, though I don't think we have a ticket for the dashboard yet

Reply:
Added the dashboard ticket to the TODO.